.flake8

```diff
@@ -0,0 +1,36 @@
+[flake8]
+exclude =
+max-line-length = 88
```

Comment:
In Intellij we generally use 120, do you have any reference library?

Reply:
I think this is partially inspired from [ray's flake8 file](https://github.com/ray-project/ray/blob/master/.flake8)?

Reply:
Yes it is copied verbatim from Ray's flake8 file

Reply:
+1 to maintaining a consistent style with Ray. I also prefer shorter lines (e.g. 70-90 characters) since it makes it easier to develop/review multiple files side-by-side, is helpful to anyone w/o great eyesight who reads the code (since they can enlarge the text while still fitting whole lines on their monitor), and encourages writing more diff-friendly code that is more (e.g. by forcing you to not pack too many arguments to a function on one line of code).